transport/src/test/java/io/netty/channel/BaseChannelTest.java

```diff
@@ -22,19 +22,18 @@
 import io.netty.buffer.Unpooled;
 import io.netty.channel.local.LocalChannel;
 import io.netty.channel.local.LocalServerChannel;
+import org.junit.After;
+import org.junit.Before;
 
 import static org.junit.Assert.*;
 
-class BaseChannelTest {
+public class BaseChannelTest {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add some tests to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2187)